plugin/RealmPlugin/NSFileManager+GlobAdditions.h

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import <Foundation/Foundation.h>
+
+@interface NSFileManager (GlobAdditions)
```

Comment:
No. They are just only conveniences. 

They are written as categories because the existing interfaces are too clumsy to work with and they are just extensions of their respective classes. I thought it would be nice to group things in their responsibilities so I wrote it that way.

If you feel strongly against this, I can change them into static functions in `RLMPRealmPlugin` itself.